Tests/Asm/Ansible/Command/AnsibleGalaxyTest.php

```diff
@@ -36,8 +34,11 @@ public function testCreateInstance()
      */
     public function testExecute(AnsibleGalaxyInterface $command)
     {
-        if (Env::isWindows())
-            $this->markTestSkipped('Skipped on Windows');
+        // Skipped on Windows
```

Comment:
same thing here: It should be clearly visible that there's no test for windows env here. Everything else might lead to wrong assumptions

Reply:
I had conflicting feelings about this. That code is mostly because there is no chance to run on Windows and wanted to avoid the all the notifications which looks a bit like errors in PhpStorm. Mostly not to get distracted by useless notification but I see your point.